time timeStamp
float64[] torques